recipe/run_test.sh

```diff
@@ -27,7 +34,8 @@ if [[ $PKG_NAME == "openmpi" ]]; then
   $MPIEXEC --help
   $MPIEXEC -n 4 ./helloworld.sh
 
-  test -f $PREFIX/include/mpi.mod
+# test -f $PREFIX/include/mpi.mod
```

Comment:
make sure to put this test back. If it fails, something is broken that needs to be fixed.